content/resources/Reference/Framework_versions.md

```diff
@@ -18,7 +18,7 @@ Babylon comes in two different versions - the preview version and the stable ver
 
 Both versions are considered to be stable and can be used in production. In certain cases (if we still haven't introduced any new features we don't want to have on the stable branch) the preview and the stable versions are identical.
 
-Both versions have their own CDN endpoint. The preview version is available on [preview.babylonjs.com](https://preview.babylonjs.com) and the stable version is available on [cdn.babylonjs.com](https://cdn.babylonjs.com).
+Both version editions can be found on [Github](https://github.com/BabylonJS/Babylon.js#cdn).

```

Comment:
I should have clarified, this is to cleanup any potential confusion. Users need to know that there are "different" available Babylon.js CDN versions, to prevent projects breaking after a major update or such.

Reply:
I am not sure why it is needed? What is the difference in that case?

Reply:
You should fetch from master, as this issue was updated with the "xxx" part of things.
There is no version in the cdn. There is a piece cdn, and a stable cdn. Evidently they are the same at the moment, but might be different in the future. The link you provide days the same thing. I am really unsure why we need to link an outside source (not in this page) to explain the same thing